test/tests/ver-1e/comparison_ver-1e.py

```diff
@@ -4,28 +4,50 @@
 from matplotlib import gridspec
 import pandas as pd
 from scipy import special
+from numpy import sin,tan,sqrt,exp
 
 # ========= Comparison of concentration as a function of time ===================
 
 fig = plt.figure(figsize=[6.5, 5.5])
 gs = gridspec.GridSpec(1, 1)
 ax = fig.add_subplot(gs[0])
 
-tmap_sol = pd.read_csv("./ver-1e_csv.csv")
+tmap_sol = pd.read_csv("./gold/ver-1e_csv.csv")
 tmap_time = tmap_sol['time']
 tmap_conc = tmap_sol['conc_point1']
 
-analytical_sol = pd.read_csv("./analytical_time.csv")
-analytical_time = analytical_sol['t']
-analytical_conc = analytical_sol['u']
+#Analytical solution
+t = np.expand_dims(tmap_time,axis=0)
+x = 15.75e-6
```

Comment:
```suggestion
x = 15.75e-6           # distance into the SiC layer of a PyC/SiC composite structure (microns)   
```

Reply:
Actually the distance is in meters here not microns

Reply:
Good catch, @RemDelaporteMathurin !